survlimepy/survlime_explainer.py

```diff
@@ -180,6 +180,7 @@ def plot_weights(
         scale_with_data_point: bool = False,
         figure_path: Optional[str] = None,
         with_colour: bool = True,
+        absolute_vals: bool = False,
```

Comment:
Can you please add documentation for the `absolute_vals` variable?

Reply:
Added.